packages/builder/src/components/automation/SetupPanel/RowSelector.svelte

```diff
@@ -94,17 +94,22 @@
     dispatch("change", newValue)
   }
 
-  const onChangeSetting = (e, field) => {
-    let fields = {}
-    fields[field] = {
-      clearRelationships: e.detail,
+  const onChangeSetting = (e, field, setting) => {
```

Comment:
NIT: the parameters of this function look quite strange. I would expect: field, key, value. Now is the other way round, and it gets an event instead of a value.
I needed to check the usage of this function to understand what it was the usage